laws/jvm/src/main/scala/kantan/codecs/laws/discipline/ArbitraryInstances.scala

```diff
@@ -50,7 +50,9 @@ trait ArbitraryInstances extends CommonArbitraryInstances {
     port     <- Gen.choose(0, 65535)
     length   <- Gen.choose(0, 5)
     path     <- Gen.listOfN(length, genPathElement)
-  } yield new URL(protocol, host, port, path.mkString("/", "/", ""))
+  } yield
+  // public URI(String scheme, String userInfo, String host, int port, String path, String query, String fragment)
```

Comment:
I would prefer using named parameter, if the purpose of this comment is to document the very string heavy call below:
```scala
new URI(scheme = protocol, userInfo = "", host = host....
```

Reply:
Does it work on Java API?

Reply:
Huh. I had no idea named parameters did not work on Java APIs, but you're right, they don't! I assumed the language merely didn't have the syntax, but no, apparently the name is also anonimized in the bytecode...